NOTICES.md

```diff
@@ -10,3 +10,105 @@ open source packages that are attributed below in this notice. Where required,
 copies of the license agreement are provided in separate files in the Licenses
 subdirectory. Unless noted below, all open source software is distributed in its
 original form without modification.
+
+## Dripicons
+
+https://github.com/amitjakhu/dripicons
+
+Copyright (c) Amit Jakhu, some rights reserved. Freely used without modification
+under terms of the [CC BY-SA
+4.0](https://creativecommons.org/licenses/by-sa/4.0/) license; please see the
+[full license](https://creativecommons.org/licenses/by-sa/4.0/legalcode) for
+warranty disclaimer notifications. [Original source:
+amitjakhu/dripicons](https://github.com/amitjakhu/dripicons).
+
+## Work Sans Font
+
+https://github.com/weiweihuanghuang/Work-Sans
+
+Copyright (c) 2019, The Work Sans Project Authors
```

Comment:
Not absolutely critical, but might make for nicer formatting: can you add a tab (4 spaces) in front of each of lines 29 through 114?